assets/cubyz/biomes/savannah.zig.zon

```diff
@@ -0,0 +1,84 @@
+.{
+	.properties = .{
+		.dry,
```

Comment:
It doesn't look that dry to me, there are many trees and grass after all.
Also it probably should be hot, no?

Reply:
Well the point is that for this PR you'd only need to touch those biomes to start with.
Here you could probably give the desert and ferrock mountains a barren property, and the savannah could take their spot in the hot and dry and section.
And they really should be hot, to avoid situations like this:
![Screenshot at 2025-01-10 19-19-17](https://github.com/user-attachments/assets/b0a687d9-70f5-4c48-921a-2a4864d691bf)